test/gtest/uct/uct_test.cc

```diff
@@ -1448,7 +1448,7 @@ ucs_status_t uct_test::send_am_message(entity *e, uint8_t am_id, int ep_idx)
 void test_uct_iface_attrs::init()
 {
     uct_test::init();
-    m_e = uct_test::create_entity(0);
+    m_e = uct_test::create_entity(NULL, NULL, NULL, NULL);
```

Comment:
maybe we should rename create_entity() to create_entity_tm()?
or just unite 'create_entity(size_t rx_headroom, ...' with ' create_entity(uct_tag_unexp_eager_cb_t eager_cb, ..' so all tests will pass TM callbacks by default?
it's confusing we have 4 variants of this function..